Adds Growth plan template (#647)
openjs-foundation/cross-project-council: GROWTH_PLAN_TEMPLATE.md

# ${PROJECT} Growth Plan

> _Note: the purpose of the growth plan is to provide a roadmap for a project to move from Growth stage to Impact stage._

Reviewer: Based on discussions from #650
```suggestion
_Note: the purpose of the growth plan is to provide a roadmap for a project to move from At Large stage to Impact stage._
```

Reviewer: `Growth` is mentioned couple of times more in the following paragraphs

## Growth stage requirements

_Note: in theory, Growth stage requirements should already be met by all Growth stage projects.
As requirements might have changed after a project was accepted at the Growth stage, this isn't necessarily always the case.
So making sure that all Growth requirements are met is a good first step in any Growth plan._

| Requirement | Current status | Desired status | Plan | Timeline | Completed |
| --- | ---| --- | --- | --- | --- | 

> | Successfully used in production by at least two independent end users. | {CURRENT_STATUS} | 2 | {PLAN} | {DATE} | <!--✅--> |

Reviewer: I would suggest to generalise this a bit more. Production usage might work for Node.js but not for other Node.js based tools.

```suggestion
| Project shows substantial adoption in production environments or usage within the ecosystem | {CURRENT_STATUS} | 2 | {PLAN} | {DATE} | <!--✅--> |
```

| Substancial ongoing flow of commits and merged contributions. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
| Sufficient level of community participation to meet goals outlined in growth plan. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
| Accepts security reports and publicly disclose them after the fix has been made available. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |

## Impact stage requirements

| Requirement | Current status | Desired status | Plan | Timeline | Completed |
| --- | ---| --- | --- | --- | --- | 
| Governing body of >= 5 members of which no more than 1/3 is affiliated with the same employer. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
| Publicly documented governance, decision-making, contribution, and release processes. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |

> | Healthy number of committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |

Reviewer: This suggestion might better emphasise the importance of ongoing contributions.

```suggestion
| Healthy number of regular committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
```
